tests/johnsnowlabs/test_jsl_api.py

```diff
@@ -0,0 +1,730 @@
+import json
+import os
+from pathlib import Path
+from unittest import mock
+
+import numpy as np
+import pandas as pd
+import pyspark
+import pytest
+import yaml
+from johnsnowlabs import nlp
+from packaging.version import Version
+
+import mlflow
+import mlflow.tracking
+import mlflow.utils.file_utils
+from mlflow import pyfunc
+from mlflow.environment_variables import MLFLOW_DFS_TMP
+from mlflow.models import Model, build_docker, infer_signature
+from mlflow.models.utils import _read_example
+from mlflow.pyfunc import spark_udf
+from mlflow.store.artifact.s3_artifact_repo import S3ArtifactRepository
+from mlflow.tracking._model_registry import DEFAULT_AWAIT_MAX_SLEEP_SECONDS
+from mlflow.tracking.artifact_utils import _download_artifact_from_uri
+from mlflow.utils.environment import _mlflow_conda_env
+from mlflow.utils.file_utils import TempDir
+from mlflow.utils.model_utils import _get_flavor_configuration
+from tests.helper_functions import (_assert_pip_requirements,
+                                    _compare_conda_env_requirements,
+                                    _compare_logged_code_paths,
+                                    _mlflow_major_version_string,
+                                    score_model_in_sagemaker_docker_container)
+
+# TODO you must update this
+license_keys = {
+    "AWS_ACCESS_KEY_ID": None,
+    "AWS_SECRET_ACCESS_KEY": None,
+    "SPARK_NLP_LICENSE": None,
+    "SECRET": None,
+}
```

Comment:
This can lead to mistakenly pushing secrets to GitHub. Let's remove this (unless we want to leak them).